facebook-commerce.php

```diff
@@ -1017,6 +1020,22 @@ private function should_update_visibility_for_product_status_change( $new_status
 		return ( $old_status === 'publish' && $new_status !== 'publish' ) || ( $old_status === 'trash' && $new_status === 'publish' ) || ( $old_status === 'future' && $new_status === 'publish' );
 	}
 
+	/**
+	 * Deletes a product from Facebook when status is changed to draft.
+	 *
```

Comment:
Missing `@since` documentation element. You can use it with `x.x.x` which will be automatically expanded into proper plugin version by the release script.

```suggestion
	 *
	 * @since x.x.x
```